File: COMP_HW1/COMP_HW1_java/Work.java
public class Work extends Person{
	private String team;
	
	public Work(String a, String b, int c, String t) {
		super(a, b, c);
		team = t;
	}
	
	public void setTeam(String t){
		team = t;
	}
	
	public String getTeam(){
		return team;
	}
	
	public void print(){
		super.print();
		System.out.print("_" + team);
	}
	
}

File: COMP_HW1/COMP_HW1_cpp/Work.cpp
#include "Work.h"
using namespace std;

Work :: Work(string a, string b, int c, string t) : Person(a, b, c)
{
	team = t;
}


void Work::setTeam(string t){
	team = t;
}

string Work::getTeam(){
	return team;
}

void Work::print(){
	Person::print();
	cout << "_" << team;
}


